test/cases/dump/dump.sql

```diff
@@ -0,0 +1,34 @@
+DROP DATABASE IF EXISTS `t`;
+CREATE DATABASE `t`;
+USE `t`;
+create table t1(c1 int primary key auto_increment,c2 tinyint not null default 4,c3 smallint,c4 bigint,c5 tinyint unsigned,c6 smallint unsigned,c7 int unsigned,c8 bigint unsigned,c9 float,c10 double,c11 date,c12 datetime,c13 timestamp on update current_timestamp,c14 char,c15 varchar,c16 json,c17 decimal,c18 text,c19 blob,c20 uuid);
+insert into t1 values(1,1,1,1,1,1,1,1,1,1,'2019-01-01','2019-01-01 00:00:00','2019-01-01 00:00:00','a','a','{"a":1}','1212.1212','a','aza','00000000-0000-0000-0000-000000000000');
+select * from t1;
+modump database t into 'a.sql';
```

Comment:
maybe you can test dump with source clause?

Reply:
`source clause` cannot be parsed by bvt